runtime/src/prioritization_fee.rs

```diff
@@ -0,0 +1,358 @@
+use {
+    crate::transaction_priority_details::GetTransactionPriorityDetails,
+    solana_measure::measure,
+    solana_sdk::{
+        clock::Slot,
+        pubkey::Pubkey,
+        saturating_add_assign,
+        transaction::{SanitizedTransaction, MAX_TX_ACCOUNT_LOCKS},
+    },
+    std::collections::HashMap,
+};
+
+#[derive(Debug, Default)]
+struct PrioritizationFeeMetrics {
+    // Count of writable accounts in slot; including accounts have minimum prioritization fee equal
+    // to blocker's minimum fee, the number of these irrelevant accounts is the difference between
+    // this counter and `relevant_writable_accounts_count`.
+    total_writable_accounts_count: u64,
+
+    // Count of writeable accounts with a minimum prioritization fee higher than the minimum transaction
+    // fee for this slot.
+    relevant_writable_accounts_count: u64,
+
+    // Total prioritization fees included in this slot.
+    total_prioritization_fee: u64,
+
+    // Accumulated time spent on tracking prioritization fee for each slot.
+    total_update_elapsed_us: u64,
+}
+
+impl PrioritizationFeeMetrics {
+    fn increment_total_prioritization_fee(&mut self, val: u64) {
```

Comment:
Just a naming nit.. I think of "increment" as being "add one", rather than accumulating an arbitrary value (here and on the line below)